WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -58,6 +61,9 @@
     private boolean mEmailFollowersFetchRequestInProgress;
     private boolean mCanRefreshEmailFollowers;
     private int mEmailFollowersLastFetchedPage;
+    private boolean mViewersEndOfListReached;
```

Comment:
Aha, I know understood what the `*EndOfListReached` variables represent! They are local flags about the _remote_ list of data. I had the impression they were related to the View state (like, the end of the RecyclerView has been reached). May I suggest adding a comment to the relevant variables to make it clear that are about the remote data? Alternatively, you might want to consider renaming them to something like "mViewersNoMoreData" or something that would hint towards the data instead of the View. Thanks!